src/services/ipfs/utils/epub.ts

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable no-bitwise */
+/* eslint-disable import/prefer-default-export */
+export function checkIfEPUB(uint8Array: Uint8Array) {
```

Comment:
Whe have 'file-type' package:
```
import { fileTypeFromBuffer } from 'file-type';
const fileType = await fileTypeFromBuffer(raw);

  return fileType?.mime || 'text/plain';
```
  
did you tired to use that?

Reply:
this is removed as redundant